### ISSUE TITLE
Panic when doing near text search when no vectorizers in the collection

### ISSUE THREAD

**Issue by rlmanrique** [weaviate/weaviate#6066]
### How to reproduce this bug?

Deploy Weaviate with three nodes using the version: stable-v1.24-8948fc5
Python weaviate-client: 4.9.0

It can be reproduced using the [weaviatest](https://github.com/weaviate/weaviatest) container:

```
docker run -it weaviatest create collection
docker run -it weaviatest create data --limit 100
docker run -it weaviatest query data --limit 100 --search_type vector
```

The container is doing internally the query:
```
        response = collection.with_consistency_level(cl).query.near_text(
            query=query,
            return_metadata=MetadataQuery(distance=True, certainty=True),
            limit=num_objects,
        )
```

The root cause it to create a collection without vectorizers and later perform a query that should use them.

### What is the expected behavior?

No panic at Weaviate. Query succesful

### What is the actual behavior?

Weaviate node processing the query reboots with a panic:

### Supporting information

```
panic: ValidateParam was called without any known params present

goroutine 6183 [running]:
github.com/weaviate/weaviate/usecases/modules.(*Provider).validateSearchParam(0x4002b16960, {0x177625c, 0x8}, {0x157b9e0, 0x4004ec6f00}, 0x40001c1970)
	/go/src/github.com/weaviate/weaviate/usecases/modules/modules.go:483 +0x2fc
github.com/weaviate/weaviate/usecases/modules.(*Provider).ValidateSearchParam(0x4002b16960, {0x177625c, 0x8}, {0x157b9e0, 0x4004ec6f00}, {0x4004ea2888?, 0x10?})
	/go/src/github.com/weaviate/weaviate/usecases/modules/modules.go:467 +0x70
github.com/weaviate/weaviate/usecases/traverser.(*nearParamsVector).validateNearParams(0x4000017280, 0x0, 0x0, 0x0?, {0x4004da3518, 0x1, 0x40?})
	/go/src/github.com/weaviate/weaviate/usecases/traverser/near_params_vector.go:119 +0x2b0
github.com/weaviate/weaviate/usecases/traverser.(*nearParamsVector).vectorFromParams(0x4000017280, {0x1c09e58, 0x4004e7d8c0}, 0x0, 0x0, 0x400582f2c0, {0x4004ea2888, 0x6}, {0x0, 0x0})
	/go/src/github.com/weaviate/weaviate/usecases/traverser/near_params_vector.go:50 +0x6c
github.com/weaviate/weaviate/usecases/traverser.(*Explorer).vectorFromParams(_, {_, _}, {0x0, {0x4004ea2888, 0x6}, 0x4003d2b500, 0x0, {0x0, 0x0, ...}, ...})
	/go/src/github.com/weaviate/weaviate/usecases/traverser/explorer.go:726 +0x44
github.com/weaviate/weaviate/usecases/traverser.(*Explorer).getClassVectorSearch(_, {_, _}, {0x0, {0x4004ea2888, 0x6}, 0x4003d2b500, 0x0, {0x0, 0x0, ...}, ...})
	/go/src/github.com/weaviate/weaviate/usecases/traverser/explorer.go:201 +0x4c
github.com/weaviate/weaviate/usecases/traverser.(*Explorer).GetClass(_, {_, _}, {0x0, {0x4004ea2888, 0x6}, 0x4003d2b500, 0x0, {0x0, 0x0, ...}, ...})
	/go/src/github.com/weaviate/weaviate/usecases/traverser/explorer.go:147 +0x104
github.com/weaviate/weaviate/usecases/traverser.(*Traverser).GetClass(_, {_, _}, _, {0x0, {0x4004ea2888, 0x6}, 0x4003d2b500, 0x0, {0x0, ...}, ...})
	/go/src/github.com/weaviate/weaviate/usecases/traverser/traverser_get.go:69 +0x420
github.com/weaviate/weaviate/adapters/handlers/grpc/v1.(*Service).search(0x400513b500, {0x1c09e58, 0x4004e7d8c0}, 0x4004e9f7a0)
	/go/src/github.com/weaviate/weaviate/adapters/handlers/grpc/v1/service.go:197 +0x188
github.com/weaviate/weaviate/adapters/handlers/grpc/v1.(*Service).Search.func1()
	/go/src/github.com/weaviate/weaviate/adapters/handlers/grpc/v1/service.go:170 +0x3c
github.com/weaviate/weaviate/entities/errors.GoWrapperWithBlock.func1()
	/go/src/github.com/weaviate/weaviate/entities/errors/go_wrapper.go:53 +0x70
created by github.com/weaviate/weaviate/entities/errors.GoWrapperWithBlock in goroutine 6182
	/go/src/github.com/weaviate/weaviate/entities/errors/go_wrapper.go:42 +0xa0
```

### Server Version

stable-v1.24-8948fc5

### Code of Conduct

- [X] I have read and agree to the Weaviate's [Contributor Guide](https://weaviate.io/developers/contributor-guide) and [Code of Conduct](https://weaviate.io/service/code-of-conduct)

**Comment by dirkkul** [comment]
Hi, this was fixed in stable/v1.25 - could you test that again? As stable/v1.24 is not supported anymore I would not backport the fix

**Comment by rlmanrique** [comment]
True, it is fixed in stable/v1.25.
